app/src/main/res-screen/login/layout/fragment_login_screen.xml

```diff
@@ -0,0 +1,188 @@
+            android:orientation="horizontal"
+            android:paddingLeft="10dp"
+            android:paddingRight="10dp">
+
+            <EditText
+                android:id="@+id/email_input"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:backgroundTint="@color/white"
+                android:fontFamily="@font/raleway_regular"
+                android:hint="Email"
+                android:inputType="textEmailAddress"
+                android:textColor="@color/light_purple"
+                android:textColorHint="@color/light_purple"
+                android:textSize="16dp" />
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/pass_field"
+            android:layout_width="280dp"
+            android:layout_height="40dp"
+            android:layout_below="@id/login_field"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="16dp"
+            android:background="@drawable/purple_stroke_for_fields"
+            android:gravity="center_vertical"
+            android:orientation="horizontal">
+
+            <EditText
+                android:id="@+id/pass_input"
+                android:layout_width="255dp"
+                android:layout_height="match_parent"
+                android:backgroundTint="@color/white"
+                android:fontFamily="@font/raleway_regular"
+                android:hint="Password"
+                android:paddingLeft="10dp"
+                android:paddingRight="2dp"
+                android:textColor="@color/light_purple"
+                android:textColorHint="@color/light_purple"
+                android:textSize="16dp" />
+
+            <ImageView
+                android:layout_width="16dp"
+                android:layout_height="16dp"
+                android:src="@drawable/ic_welcome_lock" />
+        </LinearLayout>
+
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/login_button"
+            android:layout_width="300dp"
+            android:layout_height="60dp"
+            android:layout_below="@id/pass_field"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="6dp"
+            android:background="@drawable/purple_gradient_for_buttons"
+            android:fontFamily="@font/raleway_regular"
+            android:text="@string/login_screen_login"
+            android:textAllCaps="false"
+            android:textColor="@color/white"
+            android:textSize="21sp" />
+
+    </RelativeLayout>
+    <LinearLayout
+        android:id="@+id/get_new_text"
+        android:layout_width="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginTop="18dp"
+        android:layout_centerHorizontal="true"
+        android:layout_below="@id/login_form"
+        android:layout_height="wrap_content">
+        <TextView
```

Comment:
Such cases when you have textviews with different styles should be handled via Spannable used on single TextView

http://old.flavienlaurent.com/blog/2014/01/31/spans/
https://medium.com/androiddevelopers/spantastic-text-styling-with-spans-17b0c16b4568
https://chiuki.github.io/advanced-android-textview/#/